tsconfig.json

```diff
@@ -13,7 +13,6 @@
     "noUnusedLocals": false,
     "noUnusedParameters": true,
     "outDir": "lib",
-    "removeComments": true,
```

Comment:
is there any way to have it do this only for types?

Reply:
Should only affect types as-is.
The resulting JS is minified with terser that already removes comments from what I can see.